/**:
  ros__parameters:
    LEFT_TRIGGER: 12
    RIGHT_TRIGGER: 13
    LEFT_STICK_VERTICAL: 1
    LEFT_STICK_HORIZONTAL: 0
    RIGHT_STICK_VERTICAL: 4
    RIGHT_STICK_HORIZONTAL: 3

    A_BUTTON: 8 # A button for Xbox, (Cross)     x button for PS
    B_BUTTON: 11 # B button for Xbox, (Square)    ⬜ button for PS
    X_BUTTON: 9 # X button for Xbox, (Circle)    ◯ button for PS
    Y_BUTTON: 10 # Y button for Xbox, (Triangle)  △ button for PS
